lib/ngx/balancer.md

```diff
@@ -156,6 +157,26 @@ all the domain names before entering the `balancer_by_lua*` handler (for example
 you can perform DNS lookups in an earlier phase like [access_by_lua*](https://github.com/openresty/lua-nginx-module#access_by_lua)
 and pass the results to the `balancer_by_lua*` handler via [ngx.ctx](https://github.com/openresty/lua-nginx-module#ngxctx).
 
+In the `http` subsystem a third argument `opts` is supported since the
+`v0.1.18` release of this library, and can contain the following options:
+
+* `pool` specify a custom name for the connection pool to use. If omitted, then
+  the connection pool name will be generated from the string template
+  `"<host>:<port>"`. Note that the connection will not be kept-alive unless
```

Comment:
@thibaultcha If I am wrong, please correct me: I think it will [use a global pool in per worker when the pool_crc32 is 0](https://github.com/openresty/lua-resty-core/pull/276/files#diff-973c4ef5939470d0a1c529e4e692db8eR169) instead of use `"<host>:<port>"` as the name for connection pool?

Reply:
https://github.com/openresty/lua-nginx-module/pull/1600/files#diff-998ae59f12e2d4f8f745561e5c824912R1052